client/src/partials/patients/record/patient_record.js

```diff
@@ -1,26 +1,61 @@
 angular.module('bhima.controllers')
 .controller('PatientRecordController', PatientRecordController);
 
-PatientRecordController.$inject = ['$stateParams', 'PatientService', 'NotifyService', 'moment'];
+PatientRecordController.$inject = ['$stateParams', 'PatientService', 'NotifyService', 'moment', 'Upload', '$timeout'];
 
-function PatientRecordController($stateParams, Patients, Notify, moment) {
+function PatientRecordController($stateParams, Patients, Notify, moment, Upload, $timeout) {
   var vm = this;
   var patientID = $stateParams.patientID;
   
   vm.loading = true;
+  vm.uploadFiles = uploadFiles;
+  vm.uploadButtonText = 'PATIENT_RECORDS.UPLOAD_PICTURE';
+ 
+  function uploadFiles(file, invalidFiles) {
+    if(invalidFiles.length){
+      Notify.danger('FORM.WARNNINGS.BAD_FILE_TYPE');  
+      return
```

Comment:
Did you run `gulp lint` on this file?  Does it pass?  You are missing a semicolon here.